lua/starfall/transfer.lua

```diff
@@ -161,6 +190,37 @@ else
 		net.SendToServer()
 	end
 
+	
+	function SF.SendError(chip, message, traceback)
+		local owner, is_blocked = chip.owner, false
+		if owner and owner:IsValid() then
+			is_blocked = SF.BlockedUsers:isBlocked(owner:SteamID())
+		end
+		net.Start("starfall_error")
+			net.WriteEntity(chip)
+			net.WriteString(string.sub(message, 1, 1024))
+			net.WriteString(string.sub(traceback, 1, 1024))
+			net.WriteBool(GetConVarNumber("sf_timebuffer_cl") > 0 and not is_blocked)
+		net.SendToServer()
+	end
+
+	net.Receive("starfall_error", function()
+		local chip = net.ReadEntity()
+		if not chip:IsValid() then return end
+		local owner = chip.owner
+		if not owner:IsValid() then return end
```

Comment:
`chip.owner` is `nil` here if the clientside doesn't get to initialize, either by erroring immediately from serverside or erroring immediately from clientside, even on your own client.